zh/starcoder2.md

```diff
@@ -6,47 +6,35 @@ authors:
 - user: loubnabnl
 - user: anton-l
 - user: nouamanetazi
-translators:
+translator:
```

Comment:
I don't know what happens with this file `zh/starcoder2.md`: when I updated my main copy, I got an issue. I think it's related to some capitalization changes in the file.